pkg/graph/resolvers/trb_request_status_test.go

```diff
@@ -0,0 +1,213 @@
+package resolvers
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/lib/pq"
+
+	"github.com/cmsgov/easi-app/pkg/appconfig"
+	"github.com/cmsgov/easi-app/pkg/email"
+	"github.com/cmsgov/easi-app/pkg/local"
+	"github.com/cmsgov/easi-app/pkg/models"
+	"github.com/cmsgov/easi-app/pkg/testhelpers"
+)
+
+// TestTRBRequestStatus tests the overall status of a TRB request
+func (s *ResolverSuite) TestTRBRequestStatus() {
+	ctx := context.Background()
+	store := s.testConfigs.Store
+
+	config := testhelpers.NewConfig()
+
+	// set up Email Client
+	emailConfig := email.Config{
+		GRTEmail:               models.NewEmailAddress(config.GetString(appconfig.GRTEmailKey)),
+		ITInvestmentEmail:      models.NewEmailAddress(config.GetString(appconfig.ITInvestmentEmailKey)),
+		AccessibilityTeamEmail: models.NewEmailAddress(config.GetString(appconfig.AccessibilityTeamEmailKey)),
+		TRBEmail:               models.NewEmailAddress(config.GetString(appconfig.TRBEmailKey)),
+		EASIHelpEmail:          models.NewEmailAddress(config.GetString(appconfig.EASIHelpEmailKey)),
+		URLHost:                config.GetString(appconfig.ClientHostKey),
+		URLScheme:              config.GetString(appconfig.ClientProtocolKey),
+		TemplateDirectory:      config.GetString(appconfig.EmailTemplateDirectoryKey),
+	}
+	localSender := local.NewSender()
+	emailClient, err := email.NewClient(emailConfig, localSender)
+	if err != nil {
+		s.FailNow("Unable to construct email client with local sender")
+	}
+
+	anonEua := "ANON"
+
+	stubFetchUserInfo := func(context.Context, string) (*models.UserInfo, error) {
+		return &models.UserInfo{
+			EuaUserID:  anonEua,
+			CommonName: "Anonymous",
+			Email:      models.NewEmailAddress("anon@local.fake"),
+		}, nil
+	}
+
+	stubFetchUserInfos := func(ctx context.Context, euaIDs []string) ([]*models.UserInfo, error) {
+		userInfos := []*models.UserInfo{}
+
+		for i, euaID := range euaIDs {
+			userInfo := &models.UserInfo{
+				EuaUserID:  euaID,
+				CommonName: strconv.Itoa(i),
+				Email:      models.NewEmailAddress(fmt.Sprintf("%v@local.fake", i)),
+			}
+			userInfos = append(userInfos, userInfo)
+		}
+
+		return userInfos, nil
+	}
+
+	trb := models.NewTRBRequest(anonEua)
+	trb.Type = models.TRBTNeedHelp
+	trb.State = models.TRBRequestStateOpen
+	trb, err = CreateTRBRequest(s.testConfigs.Context, models.TRBTBrainstorm, s.fetchUserInfoStub, s.testConfigs.Store)
+	s.NoError(err)
+
+	s.Run("status is correctly calculated as TRB tasks are performed", func() {
```

Comment:
Does it make sense to add `close` and `reopen` to this test case?

Reply:
On closer inspection, the overall status doesn't actually appear to be affected by the `state` property (i.e. `open/closed`), so maybe that's fine it's not included here.